src/L2Controller.sol

```diff
@@ -80,4 +82,66 @@ contract L2Controller is AccessControl {
         active = true;
     }
 
+    /**********************************************************************************************/
+    /*** Relayer PSM functions                                                                  ***/
+    /**********************************************************************************************/
+
+    function swapExactIn(
```

Comment:
Maybe we should have defined this more clearly, but the ALM Controller will only want to deposit or withdraw.

Should be:

`depositPSM(address asset, uint256 amount)`
`withdrawPSM(address asset, uint256 maxAmount)`